fixtures/voter/expired/short.json

```
@@ -0,0 +1 @@
+{"t":"voter","bs":"12","pr":"23","c": "1570162684"}
```

```suggestion
{"t":"voter","bs":"12","pr":"23","c": "0"}
```
As-is it lets me vote. Changing `c` to `0` seems to properly block voting with the "Expired Card" screen.